log/logsink_test.go

```diff
@@ -217,19 +218,18 @@ func TestLogger_Error_WorksWithNilError(t *testing.T) {
 }
 
 func TestLogger_V_Info(t *testing.T) {
+	// loop through log levels 1-5 and log all of them to verify that they either
+	// are or are not logged according to verbosity above
 	for verbosity := 1; verbosity < 5; verbosity++ {
-		log.SetLogLevel(verbosity)
-
-		// loop through log levels 1-5 and log all of them to verify that they either
-		// are or are not logged according to verbosity above
 		for logLevel := 1; logLevel < 5; logLevel++ {
 			obs, logger := NewObservedLogger()
+			logger.GetSink().(*log.Sink).SetVerbosity(verbosity)
 
 			logger.V(logLevel).Info("hello, world")
 
 			logs := obs.TakeAll()
 
-			shouldBeLogged := verbosity >= logLevel
+			shouldBeLogged := verbosity <= logLevel
```

Comment:
I believe this was correct as before. AFAIU the previous implementation, if the loglevel of your message (e.g. error =1) is smaller than your verbosity (e.g. debug = 5), your error message should be included in the logs. In other words you log everything up to debug messages.